packages/api/package.json

```
@@ -7,11 +7,13 @@
   "main": "dist/worker.js",
   "scripts": {
     "deploy": "wrangler publish --env production",
-    "predev": "./scripts/cli.js db --start && ./scripts/cli.js db-sql --cargo --testing --reset",
-    "dev": "miniflare --watch --debug --env ../../.env",
-    "clean": "./scripts/cli.js db --clean && rm -rf docker/compose",
-    "build": "scripts/cli.js build",
-    "test": "tsc && playwright-test \"test/**/*.spec.js\" --sw src/index.js",
+    "dev": "./scripts/start-dev.sh",
+    "dev:persist": "./scripts/start-dev-persist.sh",
+    "dev:clean": "./scripts/clean-dev-persist.sh",
+    "build": "./scripts/cli.js build",
+    "test": "./scripts/start-test-local.sh",
+    "test:docker": "./scripts/start-test.sh ",
```

```suggestion
    "test": "./scripts/start-test.sh",
    "test:docker": "./scripts/start-test-docker.sh ",
```

Can we keep same namings to avoid confusion?  We would need shell script names renamed